File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1046_basico.js
import { question } from "readline-sync";

function main() {
    const entrada = get_number();

    let Nentradas = entrada.split(` `);
    let comeco = parseInt(Nentradas[0]);
    let final = parseInt(Nentradas[1]);

    const duracao = Calcular_Duracao(comeco, final);

    print(`O JOGO DUROU ${duracao} HORA(S)`);
}

function get_number() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

function Calcular_Duracao(num1, num2) {
    let duracao_maxima = 24;
    let intervalo = num1 - num2;

    if(intervalo < 0) {
        intervalo += 24;
        let tempo_duracao = duracao_maxima - intervalo;
        return tempo_duracao;
    } else {
        let tempo_duracao = duracao_maxima - intervalo;

        return tempo_duracao;
    }
}

main();

File: Exercicios_ED_I/Bee/1046_basico.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
using namespace std;

main() {
    int a, b;
    cin >> a >> b;

    int intervalo = a - b;
    int horas_Jogadas = 0;

    if (intervalo < 0) {
        intervalo *= -1;
        horas_Jogadas = intervalo;
    } else {
        horas_Jogadas = 24 - intervalo;
    }
    cout << "O JOGO DUROU " << horas_Jogadas << " HORA(S)" << endl;
}